### PR TITLE
[Icons] Update Font Sheet

### PR DESCRIPTION
**Pull request by vincentsalucci**
## Type of change

- [ ] Bug fix
- [X] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other

## Testing requirements
* Test all icon locations to make sure a similar experience occurs (no major changes - all icons are working as expected)

## Dependencies
* [Jslib - Icons - Update Font Sheet](https://github.com/bitwarden/jslib/pull/582)

## Before you submit
- [X] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

**Review by Hinton — COMMENTED** [review]
LGTM once jslib is bumped.

File: src/index.html [hunk 1]
```diff
@@ -11,9 +11,10 @@
     <title>Bitwarden Directory Connector</title>
     <base href="" />
   </head>
+
```
Comment (Hinton) [inline src/index.html:14]: Did we intentionally add a newline here?

File: src/scss/styles.scss [hunk 2]
```diff
@@ -1,4 +1,5 @@
 ﻿@import "../../jslib/angular/src/scss/webfonts.css";
+@import "../../jslib/angular/src/scss/bwicons/styles/style.scss";
```
Comment (Hinton) [inline src/scss/styles.scss:2]: This seems weird, why are we including this twice? And not setting the base icon url.

File: src/scss/plugins.scss [hunk 3]
```diff
@@ -1,4 +1,5 @@
-$fa-font-path: "~font-awesome/fonts";
+$bwi-font-path: "../../jslib/angular/src/scss/bwicons/fonts";
+@import "../../jslib/angular/src/scss/bwicons/styles/style.scss";
 @import "~font-awesome/scss/font-awesome.scss";
```
Comment (Hinton) [inline src/scss/plugins.scss:3]: Do we want to keep FA here?

Comment (vincentsalucci) [reply src/scss/plugins.scss]: Removed

File: src/scss/styles.scss (hunk 2 above)
Comment (vincentsalucci) [reply src/scss/styles.scss:2]: Updated - no need for base icon url.

File: src/index.html (hunk 1 above)
Comment (vincentsalucci) [reply src/index.html]: Fixed.

**Review by Hinton — DISMISSED** [review]

**Review by Hinton — DISMISSED** [review]

**Review by Hinton — APPROVED** [review]

**Pull request merged**